### PR TITLE
add size assertion to linear map

### PR DESCRIPTION
**Pull request by mforets**

### CODE REVIEW

File: src/LinearMap.jl [hunk 1]
```diff
@@ -37,6 +37,7 @@ Return the linear map of a convex set.
 If the matrix is null, a `ZeroSet` is returned; otherwise a lazy linear map.
```
Comment (schillic) [inline src/LinearMap.jl]: Is "null" a common term?

File: src/LinearMap.jl [hunk 2]
```diff
@@ -37,6 +37,7 @@ Return the linear map of a convex set.
...
 """
 function *(M::AbstractMatrix, X::LazySet)
+    @assert dim(X) == size(M, 2) "a linear map of size $(size(M)) cannot be applied to a set of dimension of $(dim(X))"
```
Comment (schillic) [inline src/LinearMap.jl]: Is "of dimension of" correct?

File: src/LinearMap.jl (hunk 1 above)
Comment (mforets) [reply src/LinearMap.jl]: as a matrix whose entries are all zero, see the wikipedia article [Zero matrix](https://en.wikipedia.org/wiki/Zero_matrix). so i could use that, or "If all the entries of the matrix are zero, ..."

File: src/LinearMap.jl (hunk 2 above)
Comment (mforets) [reply src/LinearMap.jl]: thanks, it's a typo 👍

File: src/LinearMap.jl (hunk 1 above)
Comment (schillic) [reply src/LinearMap.jl]: I am fine with "null".

**Pull request merged**